assets/src/blocks/GuestBook/GuestBookEditorScript.js

```diff
@@ -0,0 +1,25 @@
+import {frontendRendered} from '../frontendRendered';
```

Comment:
I'm not sure if I understand this 🤔 
Why do we move the block logic to an script file? Shouldn't respect the same structure as the other blocks? I mean, the script to one side, and the block logic to another side

Reply:
A lot of our blocks have this same structure 🤔 One script for the frontend and one script for the registration

Reply:
Yeah I know, but my point is, shouldn't the registration be into the `GuestBookBlock.js` instead of `GuestBookScript.js`file?

Reply:
It's in `GuestBookEditorScript.js`, I did that to be consistent with other blocks from our webpack config

Reply:
Another possibility would have been to declare the registration as a function in `GuestBookBlock.js` and then call it in `GuestBookEditorScript.js` (similar to the [CarouselHeader block](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/main/assets/src/blocks/CarouselHeader/CarouselHeaderEditorScript.js)) but I personally don't really see the point in creating 2 separate files for this

Reply:
Okay, following our conversation on Slack I understand that this is easier to review so I've updated the code